include/swift/Option/Options.td

```diff
@@ -466,4 +466,8 @@ def sanitize_EQ : CommaJoined<["-"], "sanitize=">,
   Flags<[FrontendOption, NoInteractiveOption]>, MetaVarName<"<check>">,
   HelpText<"Turn on runtime checks for erroneous behavior.">;
 
+def sanitize_coverage_EQ : Joined<["-"], "sanitize-coverage=">,
+  Flags<[FrontendOption, NoInteractiveOption]>,
+  MetaVarName<"<coverage_type>">, HelpText<"Specify sanitize coverage type">;
```

Comment:
Nitpick: I'm not super happy with this help text. At the very least it should be "sanitizer coverage", right? But even then it doesn't explain what the flag is. I'm not 100% sure what it is myself, but maybe something like "Track coverage of sanitized code".

(I'd also change the MetaVarName to just "type", because "coverage" is already in the flag name.)